.travis.yml

```diff
@@ -53,6 +53,7 @@ jobs:
                 - ./check_trailing_spaces.sh || travis_terminate 1
                 - if [ -n "$COMMIT_SCA_FILES" ]; then ./dev-tools/vendor/bin/phpmd `echo "$COMMIT_SCA_FILES" | grep -Ev "^tests/Fixtures" | xargs | sed 's/ /,/g'` text phpmd.xml || travis_terminate 1; fi
                 - ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file=.composer-require-checker.json
+                - composer require --dev localheinz/composer-normalize:0.5.0 && composer normalize --dry-run
```

Comment:
@keradus 

Not sure if this is what you like, I can imagine you would prefer me to add `localheinz/composer-normalize` to `/dev-tools/composer.json`, is that so?

If so I need to make some adjustments to `localheinz/composer-normalize` to allow for pointing to `composer.json` in a different directory.

Reply:
I was thinking about adding it to `/dev-tools/composer.json` indeed
would that be a big change ?